web/src/pages/Administration/UserManagement/UserList.js

```diff
@@ -0,0 +1,176 @@
+import { InternalUserTableData } from './Data';
+import Filters from '../../../components/Filter/filter';
+import { COMPONENTS } from '../../../utils/constants';
+import './UserList.scss';
+
+const TabPanel = (props) => {
+  const { children, value, index, ...other } = props;
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`scrollable-auto-tabpanel-${index}`}
+      aria-labelledby={`scrollable-auto-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+};
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  };
+}
+
+function UserList() {
+  const [value, setValue] = useState(0);
+  const [internalUserData, setInternalUserData] = useState(InternalUserTableData);
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  const columnForInternalUser = [
+    { field: 'id', header: 'ID', sortable: true, filter: true },
+    { field: 'name', header: 'Name', sortable: true, filter: true },
+    { field: 'userName', header: 'Username', sortable: true, filter: true },
+    { field: 'email', header: 'Email', sortable: true, filter: true },
+    { field: 'phone', header: 'Phone', sortable: true, filter: true },
+    { field: 'displayName', header: 'Display name', sortable: true, filter: true },
+    { field: 'shortName', header: 'Short name', sortable: true, filter: true }
+  ];
+  const headCellsType = ['NONE', 'NONE', 'NONE', 'NONE', 'NONE', 'TEXTFIELD', 'NONE'];
+  const numericFields = ['id', 'userName', 'phone', 'shortName'];
+
+  const { TEXT_FIELD, AUTOCOMPLETE } = COMPONENTS;
+  const FILTER_COMPONETS = [
+    {
+      control: TEXT_FIELD,
+      groupStyle: { marginLeft: '0.5rem', marginRight: '0.5rem' },
+      key: 'userName',
+      label: 'User Name',
+      placeholder: 'User Name'
+    },
+    {
+      control: TEXT_FIELD,
+      groupStyle: { marginLeft: '0.5rem', marginRight: '0.5rem' },
+      key: 'email',
+      label: 'Email',
+      placeholder: 'Email'
+    }
+  ];
+  const getFilterData = (data) => {
+    console.log('Filtered data: ', data);
+  };
+  const getFilterDataPayloadChange = (key, val) => {
+    console.log(key, val);
+  };
+
+  return (
+    <Grid className="user_list_main_cls">
+      <Grid container spacing={3}>
+        <Grid item xs={12}>
+          <Tabs
+            variant="fullWidth"
+            value={value}
+            onChange={handleChange}
+            aria-label="User Management"
+            TabIndicatorProps={{
+              style: {
+                display: 'none'
+              }
+            }}
+          >
+            <Tab
+              label={
+                <Button style={{ backgroundColor: '#94b1bf', boxShadow: 'none' }} fullWidth variant="contained">
+                  Internal User
```

Comment:
Mumtilingual is missing, please refactor all the static labels and use them from translation files